src/styles/title.scss

```
@@ -0,0 +1,25 @@
+@import "./variables.scss";
+
+.title {
+  &__label {
+    text-align: center;
+    font-size: 48px;
+    line-height: 48px;

```

Заголовки за мокапом трохи товстіші якщо це не твоє рішення..
```suggestion
  &__label {
    text-align: center;
    font-size: 48px;
    line-height: 48px;
    font-weight: 600;

```